### PR TITLE
[ARVIR] Design sweep

### PR DESCRIPTION
**Pull request by orta**
* Bumps the version to 4.1.0
* Bunch of ARVIR changes
* Analytics
* CI changes

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Note to self - greens are fake

File: fastlane/Fastfile [hunk 1]
```diff
@@ -71,49 +75,49 @@ lane :ship_beta do
...
+  #                           rewrite: true
+  #   puts "Uploaded Emission.js.map for #{project_slug}"
+  # end
+
+  # sentry.call('eigen')
   # sentry.call("eigen-staging")
 
   # Make dSYM archive available in case we failed to put them in the right place in Sentry.
-  FileUtils.mv(dsym_archive, ENV['CIRCLE_ARTIFACTS'])
+  # FileUtils.mv(dsym_archive, ENV['CIRCLE_ARTIFACTS'])
```
Comment (orta) [inline fastlane/Fastfile]: I'm ok with this PR shipping without this for today

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5156,12 +5178,16 @@
...
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EBGaramond12-Regular.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/texgyreadventor-bold.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/texgyreadventor-regular.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Bold.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-BoldItalic.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Italic.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Regular.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Semibold.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AVG65lig.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-Italic.otf",
```
Comment (mennenia) [inline Artsy.xcodeproj/project.pbxproj]: Unica suppooort

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 3]
```diff
@@ -921,6 +934,44 @@ - (void)setupAnalytics
...
+                                    @"owner_slug": controller.config.artworkSlug ?: @"unknown",
+                                };
+                            }
+                        },
+                        @{
+                            ARAnalyticsEventName: @"success",
+                            ARAnalyticsSelectorName: ARAnalyticsSelector(hasPlacedArtwork),
+                            ARAnalyticsProperties: ^NSDictionary *(ARAugmentedVIRViewController *controller, NSArray *parameters) {
+                                return @{
+                                    @"action_name" : @"placed artwork",
```
Comment (mennenia) [inline Artsy/App/ARAppDelegate+Analytics.m]: 🙌 analytiiiics

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 4]
```diff
@@ -921,6 +934,44 @@ - (void)setupAnalytics
...
+                            ARAnalyticsEventName: @"ar_view_in_room_time",
+                            ARAnalyticsSelectorName: ARAnalyticsSelector(exitARContext),
+                            ARAnalyticsProperties: ^NSDictionary *(ARAugmentedVIRViewController *controller, NSArray *parameters) {
+                                return @{
+                                    @"length" : @([controller timeInAR])
+                                };
+                            }
+                        }
+                    ]
+                },
```
Comment (mennenia) [inline Artsy/App/ARAppDelegate+Analytics.m]: Do we want to add any events for failures / problems that we want to troubleshoot? Or just keep it simple / deduct success from taps etc.

File: Artsy/View_Controllers/ARVIR/VCs/ARAugmentedVIRViewController.m [hunk 5]
```diff
@@ -309,7 +351,7 @@ - (IBAction)screenTapped:(UITapGestureRecognizer *)gesture
 
 - (IBAction)panMoved:(UIPanGestureRecognizer *)gesture
 {
-    [self.interactionController pannedOnScreen:gesture];
+//     [self.interactionController pannedOnScreen:gesture];
```
Comment (mennenia) [inline Artsy/View_Controllers/ARVIR/VCs/ARAugmentedVIRViewController.m]: ? How come this got commented out :o.

Comment (orta) [reply Artsy/View_Controllers/ARVIR/VCs/ARAugmentedVIRViewController.m]: We decided not to ship with letting a user move the artwork

File: Artsy/App/ARAppDelegate+Analytics.m (hunk 4 above)
Comment (orta) [reply Artsy/App/ARAppDelegate+Analytics.m]: Nope, just fail/success, there aren't really any intermediary states for reporting

**Pull request merged**